packages/wrangler-devtools/wrangler.toml

```diff
@@ -0,0 +1,2 @@
+name = "wrangler-devtools"
```

Comment:
Pages doesn't listen to `wrangler.toml`. You should add `--project-name=wrangler-devtools` to your `publish` command if this is the project name you are using.